### ISSUE TITLE
[RFE] output Process status in output

### ISSUE THREAD

**Issue by tinsjourney** [joknarf/pgtree#2]
it will be nice to have an option to print process status : running, sleep, defunct...

**Comment by joknarf** [comment]
see PR: https://github.com/joknarf/pgtree/pull/5

will be possible with next release:
```
# pgtree -O state -p $$
  1     (root) [systemd] S /sbin/init
  └─1929  (joknarf) [systemd] S /lib/systemd/systemd --user
    └─2712  (joknarf) [gnome-terminal-] S /usr/libexec/gnome-terminal-server
►     └─27216 (joknarf) [bash] S bash
```

**Comment by joknarf** [comment]
Now available in release : [v1.0.13](https://github.com/joknarf/pgtree/tree/v1.0.13)
